src/sass/card/card.scss

```diff
@@ -0,0 +1,135 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+}
+
+.card__container {
+    position: relative;
+    height: $card-size-default;
+}
+
+@media screen and (max-width: $_screen-size-MD) {
+    .card {
+        width: $card-size-large;
+
+        .card__container {
+            height: $card-size-large;
+        }
+    }
+}
+
+.card__content {
+    display: flex;
+    height: inherit;
+    width: inherit;
```

Comment:
Are these styles needed? They seem redundant.

Reply:
Since I'm using the `background-image` it's needed since it doesn't take up any space. I could have used an `img` tag but I was thinking we need to overlay it with other elements, i.e. the progress bar so didn't want to take up block space